src/metrics.rs

```diff
@@ -126,8 +138,10 @@ impl Metrics {
             relay_outgoing_message_latency,
             relay_outgoing_message_publish_latency,
             processed_notifications,
-            notify_latency,
             dispatched_notifications,
+            notify_latency,
```

Comment:
Fixing position, [reference to the constructor](https://github.com/WalletConnect/notify-server/pull/192/files#diff-9f90e93b5696724fd8e4407c9db0bfd381d5468bcfbf2f09c17ac8510a85e104L36)

Reply:
Nice, thanks!